src/Testable/TestContext.elm

```diff
@@ -210,3 +211,21 @@ assertCurrentModel expectedModel context =
     context
         |> currentModel
         |> Test.assertEqual (Ok expectedModel)
+
+
+{-| Assert that an outgoing port was called
+-}
+assertPortCalled : Platform.Cmd.Cmd msg -> TestContext msg model -> Assertion
+assertPortCalled expectedPort (TestContext context) =
+    case context.state of
+        Err errors ->
+            Test.fail
+                ("Expected that a port was called, but TextContext had previous errors:"
+                    ++ String.join "\n    " ("" :: errors)
+                )
+
+        Ok { effectsLog } ->
+            if containsPortCmd expectedPort effectsLog then
+                Test.assert True
+            else
+                Test.assertEqual [ expectedPort ] (EffectsLog.portCmds effectsLog)
```

Comment:
it was done this way so we can see a better output for debugging when the test fails